### ISSUE TITLE
Rendering Friend Requests requires refresh

### ISSUE THREAD

**Issue by chris-t-li** [Fantasy-Fit/fantasy-fit-web#165]
Bug when trying to complete the following:
* Login as a user;
* Search for and click on send friend request;
* Logout;
* Login as the user that received the friend request;
* Click on Friends page

The pending friend request is not there until you refresh the application.

**Comment by dkussainov** [comment]
Can we also add our sent request to pending? Not only when we receive one

**Comment by chris-t-li** [comment]
Yes, makes sense. Right now, the pending friend request only shows when you search for that friend again.
